content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,49 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The controller in the master can [bind PVCs to matching PVs](#binding) in the
+cluster. However, if you want a PVC to bind to a specific PV, you need to
```

Comment:
```suggestion
The control plane can [bind PVCs to matching PVs](#binding) in the
cluster. However, if you want a PVC to bind to a specific PV, you need to
```

Kubernetes itself does not have master nodes (some tools, like kubeadm, did have this concept but are moving to a new name).

Reply:
Thanks!